MTGToolbox.Abstract/ICardRepository.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+
+namespace MTGToolbox.Abstract
```

Comment:
I moved all of the interfaces into a new .NET Standard project. It'll help with decoupling libraries.